Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "fluvio-future"
-version = "0.6.0"
+version = "0.7.0"
```

Comment:
should be 0.6.1

Reply:
but semver.org says new feature = minor

Reply:
only for release version (> 1.0.0) otherwise bumping to 0.7 is considered breaking change which this is not

Reply:
ok, changed to 6.1